.gitignore

```diff
@@ -109,3 +109,4 @@ tests/integration/data
 
 # setuptools_scm generated version file
 bentoml/_version.py
+pyminifier
```

Comment:
add Pyminifier to dev dependencies?

Reply:
ugh no, its a python2 dependencies. I have a scripts to do that under `./scripts/tools/stubs_cleanup.sh`

Reply:
looking through pyminifier's GitHub issues, I'm not so confident about using it, could remove it from this PR?